README.md

```diff
@@ -0,0 +1,40 @@
+# Vesktop Flatpak
+
+<!-- This flatpak is based on xyz.armcord.ArmCord @ https://github.com/flathub/xyz.armcord.ArmCord -->
+
+This is the flatpak for Vesktop. Due to sandbox limitations, it might not function as expected, so some ways to get it working are detailed below:
+
+## Tray icons
+
+This flatpak has the appropriate permissions for tray icons out of the box; however, GNOME does not provide native tray icons support out of the box, due to the current specification being _horribly_ outdated and not being sandbox-friendly.
```

Comment:
this user documentation will be seen by no one.

Reply:
that's ok. we will also inform users about it on our website/faq. i don't think having this in the application description makes much sense